docs/config.html

```diff
@@ -348,6 +357,88 @@ <h3 class="mt1 f6 lh-title" id="parse.gitfunctions">GitFunctions</h3>
   </ul>
 </section>
 
+<section class="mt4">
+  <h2 id="plugindefinition" class="title-2">[PluginDefinition]</h2>
```

Comment:
maybe let's mark these sections as experimental so we can break them if we need to?